examples/arrays/arrays.out

```diff
@@ -4,6 +4,11 @@ $ ballerina run arrays.bal
 0
 1
 8
+Reversed: 8 5 6 7 4 2 3 1
```

Comment:
Is this output correct? After the change to the values was reverted?

Reply:
Fixed.